components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/model/FilterQueryBuilder.java

```diff
@@ -97,22 +100,35 @@ public String getFilterQuery() {
     }
 
     /**
-     * Get the value of hasSubAttribute.
+     * Get filter query builder meta attribute placeholders.
+     *
+     * @return List of filter query builder meta attribute placeholders.
+     */
+    public List<String> getMetaAttributePlaceholders() {
+
+        return metaAttributePlaceholders;
+    }
+
+    /**
+     * Generate and add a filter query builder meta attribute placeholder.
      *
-     * @return boolean value indicating if the filter contains a sub attribute.
+     * @return Next meta attribute placeholder.
      */
-    public boolean getHasSubAttribute() {
```

Comment:
Don't remove any public methods, it might break other codebases that use this method. Deprecate the previous method and add this as a new method. Check other places as well

Reply:
This method was added specifically for a recent feature I'm working on, and it has not yet gone to PROD. And the change is less than a week old. The current only usage is in this repo, and all places have been altered to reflect this change.